browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -644,6 +654,19 @@ void RewardsDOMHandler::GetContributionList(const base::ListValue *args) {
   }
 }
 
+void RewardsDOMHandler::GetAdsData(const base::ListValue *args) {
+  if (ads_service_ && web_ui()->CanCallJavascript()) {
+    base::DictionaryValue adsData;
+    bool ads_enabled = ads_service_->is_enabled();
+    int ads_per_hour = ads_service_->ads_per_hour();
```

Comment:
can we just call `ads_service_->GetAdsPerHour()`?

Reply:
`GetAdsPerHour` is marked as `private`:

https://github.com/brave/brave-core/blob/87b7fff6fd3694575e9693643c666a9e2baaaa1e/components/brave_ads/browser/ads_service_impl.h#L90

Reply:
can we move it to public? IMHO I don't see a point of having two getters with the same functionality

Reply:
I don't have an issue moving it to public same with `IsAdsEnabled` I just didn't want to change up the ads service implementation too much right off of the bat cc: @bridiver

Reply:
this is problematic because you're not going to see pref changes in the UI. You should be working with the prefs directly, not adding accessors for them. You should be using a PrefChangeRegistrar. See InspectUI for an example